internal/settings/settings.go

```diff
@@ -182,6 +179,7 @@ func parseArgs(args []string) (*Settings, error) {
 		"The component has to be defined in the config file and the flag has a higher precedence. "+
 		"Array config properties are overridden and maps are joined. Example --set=processors.batch.timeout=2s")
 	flagSet.BoolVar(&settings.dryRun, "dry-run", false, "Don't run the service, just show the configuration")
+	flagSet.MarkHidden("dry-run")
```

Comment:
Why don't we want to show `dry-run` in the usage? Is this experimental and can be removed?